js/popup.js

```diff
@@ -1,423 +1,396 @@
+    document.getElementById('btnQuantiModoRegister_2').onclick = btnQuantiModoRegisterClick;
 
-	document.getElementById('logo-correlate').onclick = btnQuantiModoLogoCorrelateClick;
-	document.getElementById('logo-correlate-1').onclick = btnQuantiModoLogoCorrelateClick;
-	document.getElementById('logo-correlate-2').onclick = btnQuantiModoLogoCorrelateClick;
+    document.getElementById('logo-correlate').onclick = btnQuantiModoLogoCorrelateClick;
+    document.getElementById('logo-correlate-1').onclick = btnQuantiModoLogoCorrelateClick;
+    document.getElementById('logo-correlate-2').onclick = btnQuantiModoLogoCorrelateClick;
 
-	//var optionsUrl = chrome.extension.getURL("src/options/options.html"); 
-	//var content = '<a href="' + optionsUrl + '" target="_blank">Options</a>';
+    //var optionsUrl = chrome.extension.getURL("src/options/options.html");
+    //var content = '<a href="' + optionsUrl + '" target="_blank">Options</a>';
 
 
-		
 };
 
 /**
  * Record a measurement is clicked
  */
-var onQmRcdMstButtonClicked = function()
-{
-	var name = $("#addmeasurement-variable-name").val();
+var onQmRcdMstButtonClicked = function () {
+    var name = $("#addmeasurement-variable-name").val();
 
-	if (name == '') {
-		alert("Please enter the variable name."); return;
-	}
+    if (name == '') {
+        alert("Please enter the variable name.");
+        return;
+    }
 
-	var n_value = getVariableWithName(name);
+    var n_value = getVariableWithName(name);
 
-	if ( n_value == null )
-	{
-		$("#record_a_measurement_block").hide();
-		$("#add_record_a_measurement_block").show();
-		$("#add-addmeasurement-variable-name").val(name);
+    if (n_value == null) {
+        $("#record_a_measurement_block").hide();
+        $("#add_record_a_measurement_block").show();
+        $("#add-addmeasurement-variable-name").val(name);
 
-	} 
-	else 
-	{
-		$("#record_a_measurement_block").hide();
-		$("#edt_record_a_measurement_block").show();
-		$("#edt-addmeasurement-variable-name").val(name);
+    }
+    else {
+        $("#record_a_measurement_block").hide();
+        $("#edt_record_a_measurement_block").show();
+        $("#edt-addmeasurement-variable-name").val(name);
 
-	}
+    }
 
 }
 
 
-
 // registration button clicked
-var btnQuantiModoRegisterClick = function()
-{
-	var optionsUrl = chrome.extension.getURL("options/options.html"); 
-	//var content = '<a href="' + optionsUrl + '" target="_blank">Options</a>';
+var btnQuantiModoRegisterClick = function () {
+    var optionsUrl = chrome.extension.getURL("options/options.html");
+    //var content = '<a href="' + optionsUrl + '" target="_blank">Options</a>';
 
-	chrome.tabs.create({ url: optionsUrl });
+    chrome.tabs.create({url: optionsUrl});
 };
 
 
 // registration button clicked
-var btnQuantiModoLogoCorrelateClick = function()
-{
-	chrome.tabs.create({ url: "https://app.quantimo.do/correlate/" });
+var btnQuantiModoLogoCorrelateClick = function () {

```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['btnQuantiModoLogoCorrelateClick' was used before it was defined.](https://www.codacy.com/app/m_3/QuantiModo-Chrome-Extension/file/842132528/issues/source?bid=2779649&fileBranchId=2805440#l113)